app/controllers/spree/admin/working_days_controller_decorator.rb

```diff
@@ -0,0 +1,35 @@
+class Spree::Admin::WorkingDaysController < Spree::Admin::BaseController
+  before_action :load_store
+
+  def index
+  end
+
+  def create
+    working_day = @store.working_days.new(working_day_params)
+    working_day.save
+    redirect_to admin_store_working_days_path(@store)
+  end
+
+  def edit
+    @working_day = @store.working_days.where(id: params[:id]).first
```

Comment:
why not use `find`?

Reply:
It would raise an exception if the working day doesn't exist.